.changeset/modern-insects-raise.md

```diff
@@ -0,0 +1,6 @@
+---
+"@blitzjs/auth": patch
+"blitz": patch
+---
+
+Remove unintended dependency on next-auth by removing it from the core build of @blitzjs/auth
```

Comment:
looks like there is a breaking change for anyone using the adapter, need to document this